delocate/fuse.py

```diff
@@ -83,24 +130,49 @@ def fuse_trees(to_tree, from_tree, lib_exts=(".so", ".dylib", ".a")):
                 _copyfile(from_path, to_path)
 
 
-def fuse_wheels(to_wheel, from_wheel, out_wheel):
+def fuse_wheels(
+    to_wheel: str | PathLike,
+    from_wheel: str | PathLike,
+    out_wheel: str | PathLike | None = None,
```

Comment:
If I could just throw something out there. When I used the `delocate` library to fuse wheels in a python script, rather than running the command, I just called `fuse_wheels` directly. Since I was already running python, this worked great and was easier than having to use subprocess.

With `delocate-merge` I could do the same thing. But if we move the `out_wheel` logic into the command function, it becomes a little less convenient.

I'm not sure if that is a use case you care to support, no big deal if not. But I figured I'd bring it up to see what you think.

Reply:
This suggestion was to reduce complexity of the function before the change becomes permanent, "explicit is better than implicit" and all that, so that there are fewer ambiguous contexts.

For example: Are you sure you wanted your script to use the input wheels directory for the output wheel? And not something like a temporary directory? Is the best default directory for a CLI context and API context the same directory?

I'd rather remove the default here rather than commit to an answer. You can add `out_wheel=to_wheel.parent` to replicate the current behavior in your script, it's more explicit and easier for anyone reading it to understand, and any future updates are less likely to break it.

Reply:
> Is the best default directory for a CLI context and API context the same directory?

Interesting way of putting it. I wasn't thinking of it that way.

And yes, your suggestion makes sense 👍